inclusive-design/idrc
# GH-426: Modify projects pages (resolves #426)

## Description

This pull request implements:
1. The new landing page for the "Projects" page;
2. Allow each project to have sub-pages.

## Steps to test

1. Open the deploy preview and go to "Projects";
2. The landing page should match the new design;
3. Open the admin page to author multiple pages for a project. Subpages should select "parent page title";
4. Go back to the "Projects" page and select this project. The hierarchy of pages should present in the way that's described in the https://github.com/inclusive-design/idrc/issues/426.

**Expected behavior:** 
Described above.
